Dockerfile

```
@@ -4,7 +4,7 @@ RUN apk add unzip make curl
 
 # TODO: verify GPG
 ARG TERRAGRUNT_VERSION='0.21.7'
-ARG DEFAULT_TERRAFORM_VERSION=0.12.29
+ENV DEFAULT_TERRAFORM_VERSION=0.12.29
```

We could also keep it an arg (so we can change it easily) and just assign this value to ENV
```suggestion
ARG DEFAULT_TERRAFORM_VERSION=0.12.29

ENV DEFAULT_TERRAFORM_VERSION=$DEFAULT_TERRAFORM_VERSION
```

---

> We could also keep it an arg (so we can change it easily) and just assign this value to ENV

sounds good

---

Noted and will submit the change